documentation/source/users/rmg/input.rst

```diff
@@ -909,6 +909,7 @@ all of RMG's reaction families. ::
         maximumSiliconAtoms=2,
         maximumSulfurAtoms=2,
         maximumHeavyAtoms=10,
+		maximumSurfaceSites=2,
```

Comment:
tabs instead of spaces? renders strangely on github

Reply:
yep, just realized that, should be fixed now